debug_en: true
pub_prior_pcd_en: false

num_threads: 4
num_neighbors: 10
global_leaf_size: 0.25
registered_leaf_size: 0.1
max_dist_sq: 1.0
max_iterations: 30

map_frame: map
odom_frame: odom
base_frame: base_link
lidar_frame: livox_frame

prior_pcd_file: rmuc_2025.pcd
pcd_in_topic: /cloud_registered
initialpose_topic: /initialpose

# the original transform between map and odom when mapping 
x:     8.50396
y:     4.052823
z:     0.140688
roll:  0.000000
pitch: 0.000000
yaw:   1.570796327

transform_tolerance: 0.5      
registration_frequency: 4
pose_update_frequency: 2